charts/incubator/wordpress/templates/_env.tpl

```diff
@@ -0,0 +1,40 @@
+{{/* Wordpress environment variables */}}
+{{- define "wordpress.env" -}}
+{{- $configName := printf "%s-env-config" (include "tc.common.names.fullname" .) }}
+{{- $secretName := printf "%s-env-secret" (include "tc.common.names.fullname" .) }}
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ $configName }}
+  labels:
+    {{- include "tc.common.labels" . | nindent 4 }}
+data:
+  APACHE_HTTP_PORT_NUMBER: {{ .Values.service.main.ports.main.port | quote }}
+  WORDPRESS_EMAIL: {{ .Values.wordpress.email | quote }}

```

Comment:
```suggestion
  WORDPRESS_EMAIL: {{ .Values.wordpress.email | quote }}
```
you only need `| quote` on things that maaybe can contain ONLY numbers.

On strings is not needed (not that it hurts either)